book_source/05_developer_guide/03_git/01_using-git.Rmd

```diff
@@ -29,229 +11,230 @@ easy to do.
+1. Fetch upstream remote branches
 
-There are two ways to do this. One easy way is to include the following text in your commit message:
+`git fetch upstream`
 
-* [**Github**](https://github.com/blog/1386-closing-issues-via-commit-messages)
- * to close: "closes gh-xxx" (or syn. close, closed, fixes, fix, fixed)  
- * to reference: just the issue number (e.g. "gh-xxx")
- * avoid "closes #xxx" which will cross-reference Redmine issues 
-* **Redmine**: 
- * to close: "fixes redmine #xxx" (or closes etc.) 
- * to reference: "redmine #xxx"
-* **Bitbucket**: 
- * to close: reference and use web interface!
- * to reference: "re #xxx"
+2. Checkout the correct release branch
 
-Another way is to just add the url to the issue that you are updating. For example, if you are resolving an issue in Redmine, you can simply write the text "resolved by pull request https://github.com/PecanProject/pecan/pull/1" in the comments.
+`git checkout -b release/vX.Y.Z`
 
-### For PEcAn 
+4. Compile Code with make 
 
-```
-./scripts/build.sh -c
-```
+`make`
 
-#### Committing Changes Using Pull Requests
+5. Make changes and commit them
 
-GitHub provides a useful overview of how to submit changes to a project, [Using Pull Requests](https://help.github.com/articles/using-pull-requests).
+`git add <changed_file.R>`
+`git commit -m "Describe changes"`
 
-Once you have added a feature on your local fork of the project that you would like to contribute, these are the steps:
+6. Compile and make roxygen changes
+`make`
+`make document`
 
-* Submit a Pull Request
-* Pull request is reviewed by a member of the PEcAn core group
-* Any comments should be addressed
-* Additional commits are added to the pull request
-* When ready, changes are merged
+7. Commit and push any files that were changed by make document
 
+8. Make a pull request. It is essential that you compare your pull request to the remote release branch, NOT the develop branch.
 
-#### Other Useful Git Commands:
 
+#### Link commits to issues
 
+You can reference and close issues from comments, pull requests, and commit messages. This should be done when you commit code that is related to or will close/fix an existing issue. 
+
+There are two ways to do this. One easy way is to include the following text in your commit message:
+  
+* [**Github**](https://github.com/blog/1386-closing-issues-via-commit-messages)
+* to close: "closes gh-xxx" (or syn. close, closed, fixes, fix, fixed)  
+* to reference: just the issue number (e.g. "gh-xxx")
+* avoid "closes #xxx" which will cross-reference Redmine issues 
+
+Another way is to just add the url to the issue that you are updating. For example, if you are resolving an issue in Redmine, you can simply write the text "resolved by pull request https://github.com/PecanProject/pecan/pull/1" in the comments.
```

Comment:
Is Redmine still a thing we need to be documenting?

Reply:
nope redmine is ded for PEcAn